CONTRIBUTING.md

```diff
@@ -4,58 +4,8 @@ Contributing to JuPedSim
 This project is mainly developed by a small group of researchers and students from [Jülich Research Center](http://www.fz-juelich.de/en) and [BUW](http://www.uni-wuppertal.de/).
 However you are kindly invited not only to use JuPedSim but also contributing to our open-source-project.
 It does not matter if you are a researcher, student or just interested in pedestrian dynamics.
-There are only a few rules and advices we want to give to you:
 
-- [Advice for new contributors](#advice-for-new-contributors)
-    - [First steps](#first-steps)
-    - [Workflow](#workflow)
-    - [FAQ](#faq)
-
-- [Reporting bugs and requesting features](#reporting-bugs-and-requesting-features)
-    - [Using the issue tracker](#using-the-issue-tracker)
-    - [Reporting bugs](#reporting-bugs)
-    - [Requesting features](#requesting-features)
-
-- [Writing code](#writing-code)
-    - [Coding style](#coding-style)
-    - [Unit tests](#unit-tests)
-
-- [Writing documentation](#writing-documentation)
-    - [Comments](#comments)
-    - [Documenting new features](#documenting-new-features)
-    - [Sample Templates](#sample-templates)
-
-- [Committing code](#commiting-code)
-    - [Committing guidelines](#commiting-guidelines)
-
-## Advice for new contributors
-
-### First steps
-* Clone/fork our github/gitlab repository
-~~~
-git clone https://cst.version.fz-juelich.de/jupedsim/jpscore.git
-~~~
-
-* Change to the developement branch and create a branch with your feature.
-~~~
-git checkout developement 
-git checkout -b feature_name
-~~~
-
-* Assuming you are in the jpscore folder type 
-```
-mkdir build && cd build 
-cmake .. 
-```
-
-* Download all dependencies, check if cmake tells you something is missing
-    * Boost >= 1.56
-    * Qt
-    * Vtk
-    
-* If everything compiled for the first time, you are free to start
-
-### Workflow
+## Workflow
 The branches **master** and **develop** are **protected**. You can **only push to your feature-branch.**
```

Comment:
develop branch? I think we don't use develop..

Reply:
removed develop, but i think i have to rewrite the whole paragraph.